tests/setup.ts

```diff
@@ -1,6 +1,27 @@
 import '@testing-library/jest-dom/extend-expect';
+let windowSpy: jest.SpyInstance;
 
 if (global.window) {
-  const { getComputedStyle } = window;
-  window.getComputedStyle = (elt) => getComputedStyle(elt);
+  // need to mock this function to pass default values as 0 so that we can avoid NaN warning in some of components
+  const originalGetComputedStyle = window.getComputedStyle;
+
+  const getComputedStyle = (...args: any[]) => {
+    const cssStyleDeclaration = originalGetComputedStyle(args[0], args[1]);
+    cssStyleDeclaration.setProperty('padding-left', '0');
+    cssStyleDeclaration.setProperty('padding-right', '0');
+    cssStyleDeclaration.setProperty('padding-top', '0');
+    cssStyleDeclaration.setProperty('padding-bottom', '0');
+    cssStyleDeclaration.setProperty('margin-left', '0');
+    cssStyleDeclaration.setProperty('margin-right', '0');
+    cssStyleDeclaration.setProperty('margin-top', '0');
+    cssStyleDeclaration.setProperty('margin-bottom', '0');
+    cssStyleDeclaration.setProperty('border-left-width', '0');
+    cssStyleDeclaration.setProperty('border-right-width', '0');
+    cssStyleDeclaration.setProperty('border-top-width', '0');
+    cssStyleDeclaration.setProperty('border-bottom-width', '0');
+    return cssStyleDeclaration;
```

Comment:
How does this relate to this change? https://github.com/project44/manifest/pull/471#discussion_r1379123901

Reply:
@dmorton-p44 This fixes local warnings we are getting in tests of manifest code base. To fix it on consumer side either we need to make change to default those value to 0, or we need to do same mocking on consumer side as well ( which doesn't makes sense as consumer should be fixing anything on library side ).